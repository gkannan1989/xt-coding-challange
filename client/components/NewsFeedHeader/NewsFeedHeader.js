import React from 'react';

const NewsFeedHeader = () => (
  <div className="news-feed-header">
    <table
      cellPadding="0"
      cellSpacing="0"
      width="100%"
      border="0"
      className="desktop-table"
    >
      <tr>
        <td valign="middle" align="center" className="td-header-cloumn-width">
          <span className="header-label">Top</span>
        </td>
        <td valign="middle" align="center" className="td-header-cloumn-width">
          <span className="header-label" />
        </td>
        <td valign="middle" align="center" className="td-header-cloumn-width">
          <span className="header-label" />
        </td>
        <td valign="middle">
          <span className="header-label" />
        </td>
      </tr>
    </table>
    <table
      cellPadding="0"
      cellSpacing="0"
      width="100%"
      border="0"
      className="mobile-table"
    >
      <tr>
        <td
          valign="middle"
          align="center"
          className="td-mobile-header-cloumn-width"
        >
          <span className="mobile-header-label" />
        </td>
        <td
          valign="middle"
          align="center"
          className="td-mobile-header-cloumn-width"
        >
          <span className="mobile-header-label" />
        </td>
        <td valign="middle">
          <span className="mobile-header-label" />
        </td>
      </tr>
    </table>
  </div>
);

export default NewsFeedHeader;
