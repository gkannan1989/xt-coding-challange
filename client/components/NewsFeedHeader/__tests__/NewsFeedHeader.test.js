import React from 'react';
import renderer from 'react-test-renderer';
import NewsFeedHeader from '../NewsFeedHeader.js';

it('renders', () => {
  const component = renderer.create(<NewsFeedHeader />).toJSON();
  expect(component).toMatchSnapshot();
});
