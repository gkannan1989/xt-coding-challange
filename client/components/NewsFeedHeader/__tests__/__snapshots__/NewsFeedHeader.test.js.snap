// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="news-feed-header"
>
  <table
    border="0"
    cellPadding="0"
    cellSpacing="0"
    className="desktop-table"
    width="100%"
  >
    <tr>
      <td
        align="center"
        className="td-header-cloumn-width"
        valign="middle"
      >
        <span
          className="header-label"
        >
          Top
        </span>
      </td>
      <td
        align="center"
        className="td-header-cloumn-width"
        valign="middle"
      >
        <span
          className="header-label"
        />
      </td>
      <td
        align="center"
        className="td-header-cloumn-width"
        valign="middle"
      >
        <span
          className="header-label"
        />
      </td>
      <td
        valign="middle"
      >
        <span
          className="header-label"
        />
      </td>
    </tr>
  </table>
  <table
    border="0"
    cellPadding="0"
    cellSpacing="0"
    className="mobile-table"
    width="100%"
  >
    <tr>
      <td
        align="center"
        className="td-mobile-header-cloumn-width"
        valign="middle"
      >
        <span
          className="mobile-header-label"
        />
      </td>
      <td
        align="center"
        className="td-mobile-header-cloumn-width"
        valign="middle"
      >
        <span
          className="mobile-header-label"
        />
      </td>
      <td
        valign="middle"
      >
        <span
          className="mobile-header-label"
        />
      </td>
    </tr>
  </table>
</div>
`;
